src/durableorchestrationcontext.ts

```diff
@@ -31,13 +31,14 @@ import {
 import moment = require("moment");
 import { ReplaySchema } from "./replaySchema";
 import { CallHttpOptions, Task, TimerTask } from "./types";
+import * as types from "./types";
 import { SignalEntityAction } from "./actions/signalentityaction";
 
 /**
  * Parameter data for orchestration bindings that can be used to schedule
  * function-based activities.
  */
-export class DurableOrchestrationContext {
+export class DurableOrchestrationContext implements types.DurableOrchestrationContext {

```

Comment:
so there's two `DurableOrchestrationContext` types now? I'm a bit confused by that.

Reply:
No, this file hosts the implementation of the `DurableOrchestrationContext`, which `implements` the `DurableOrchestrationContext` _declaration_ in the `types` directory. Think of it as declaration/interface vs implementation. The plan is that the class declaration defined in the `types` directory will become our public "contract" for the API provided by the `DurableOrchestrationContext`, which _excludes_ things like private fields and methods, otherwise included in the automatically generated `.d.ts` files